CefSharp.Core.Runtime/Internals/ClientAdapter.h

```diff
@@ -70,8 +70,7 @@ namespace CefSharp
 
                 CloseAllPopups(true);
 
-                //this will dispose the repository and cancel all pending tasks
-                delete _pendingTaskRepository;
```

Comment:
`delete` will call `Dispose` if it's implemented. Guess it's semantics, might be worth having pending task repository implement `IDisposable`.